### PR TITLE
Avoid autofill via check for computed background color;

### PR DESCRIPTION
**Pull request by seangates**
Hey, guess what? Chrome's computed color gives away the secret that there is some autofilled content in there. And, you must wait for the window's load event to complete because the autofilled content won't be there until the very end.

Yeah. I know. 😞 

But hey, it works!

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -34,9 +34,13 @@ module.exports = class {
         this.textboxEl.addEventListener('blur', this._onBlurListener);
         this.textboxEl.addEventListener('focus', this._onFocusListener);
 
-        if (!hasValue(this.textboxEl)) {
-            this.labelEl.classList.add(this.options.labelElementInlineModifier);
-        }
+        // check for computed background color because of Chrome autofill bug
+        window.addEventListener('load', function() {
```
Comment (ianmcburnie) [inline src/index.js]: @seangates If you don't mind, could you please change this to match the style of the other listeners? Just for consistency.

File: index.js [hunk 2]
```diff
@@ -42,9 +42,13 @@ module.exports = function () {
         this.textboxEl.addEventListener('blur', this._onBlurListener);
         this.textboxEl.addEventListener('focus', this._onFocusListener);
 
-        if (!hasValue(this.textboxEl)) {
-            this.labelEl.classList.add(this.options.labelElementInlineModifier);
-        }
+        // check for computed background color because of Chrome autofill bug
```
Comment (ianmcburnie) [inline index.js]: Also, can you please provide the reference URL for this fix/workaround in the comments? (so we remember why we did this in future!)

Comment (seangates) [reply index.js]: Sure.

**Review by ianmcburnie — APPROVED** [review]
Nice one Sean! I'll merge and put out a patch release.

**Pull request merged**